.github/workflows/docker-build-test.yaml

```diff
@@ -331,7 +346,7 @@ jobs:
       FORGE_RUNNER_DURATION_SECS: 3600
       COMMENT_HEADER: forge-framework-upgrade
       FORGE_NAMESPACE: forge-framework-upgrade-${{ needs.determine-docker-build-metadata.outputs.targetCacheId }}
-      SKIP_JOB: ${{ needs.file_change_determinator.outputs.only_docs_changed == 'true' }}
+      SKIP_JOB: ${{ needs.file_change_determinator.outputs.only_docs_changed == 'true' || needs.framework-upgrade-determinator.outputs.run_framework_upgrade_test == 'false' }}
```

Comment:
The simple detection may incorrectly say "skip" while testing is in fact needed.
Can we ensure this is not skipped if label `CICD:run-framework-upgrade-test` is set?

Reply:
> The simple detection may incorrectly say "skip" while testing is in fact needed.

Do you mean in the case that the label `CICD:run-framework-upgrade-test` is applied? If so, makes sense -- I'll update it so that we don't skip if the label is applied. 😄